openmetadata-ui/src/main/resources/ui/src/utils/DataQuality/TestSummaryGraphUtils.ts

```diff
@@ -45,6 +47,9 @@ export const prepareChartData = ({
   let showAILearningBanner = false;
   testCaseResults.forEach((result) => {
     const values = result.testResultValue?.reduce((acc, curr) => {
+      if (EXCLUDED_CHART_FIELDS.includes(curr.name ?? '')) {
+        return acc;
+      }
```

Comment:
can you please give me some inside here?
most probably this will be not needed.
are you seeing this data in chart?